sunpy/map/tests/test_mapbase.py

```diff
@@ -82,7 +86,10 @@ def generic_map():
         'PC2_2': 0,
         'NAXIS1': 6,
         'NAXIS2': 6,
-        'date-obs': '1970-01-01T00:00:00',
+        'date-obs': dobs.isot,
```

Comment:
Is there a reason we explicitly construct header as a dictionary here rather than using `make_fitswcs_header`? I understand this was already in place before this PR.

Reply:
this probably existed long before `make_fitswcs_header`